src/main/analytics/user-handlers.ts

```diff
@@ -2,15 +2,43 @@ import { ipcMain } from 'electron';
 
 import eventBus from '../event-bus';
 import { userLogout, userSignin, userSigninFailed } from './service';
+import { RemoteSyncManager } from '../remote-sync/RemoteSyncManager';
+import { DriveFilesDB } from '../database/driveFilesDB';
+import { DriveFoldersDB } from '../database/driveFoldersDB';
+import Logger from 'electron-log';
+import { clearRemoteSyncStore } from '../remote-sync/helpers';
+import { reportError } from '../bug-report/service';
 
 eventBus.on('USER_LOGGED_IN', () => {
-	userSignin();
+  userSignin();
+  const remoteSyncManager = new RemoteSyncManager(
+    {
+      files: new DriveFilesDB(),
+      folders: new DriveFoldersDB(),
+    },
+    {
+      fetchFilesLimitPerRequest: 50,
+      fetchFoldersLimitPerRequest: 50,
+      syncFiles: true,
+      syncFolders: true,
+    }
+  );
+
+  remoteSyncManager.onStatusChange((newStatus) => {
```

Comment:
This is for demo purposes, we will need to connect this with some UI feedback at some point